infra/tf/s3_minio/buckets.tf

```diff
@@ -4,7 +4,7 @@ resource "null_resource" "check_minio" {
 		command = <<EOL
 		until curl -sf ${var.s3_providers.minio.endpoint_external}/minio/health/ready; do
 			echo "Waiting for Minio to become reachable publicly (${var.s3_providers.minio.endpoint_external})..."
-			sleep 1
+			sleep 2
```

Comment:
Nit: Can we add a comment above this describing that this was changed, and why 2 is fine? If something runs into problems down the line, I find it helpful to have comments near magic numbers that might be causing it 🙌

Reply:
It wasn't changed to fix any bugs, just to make less log spam. Not sure it deserves a comment